services/QuillLMS/client/app/bundles/Teacher/components/progress_reports/diagnostic_insights/interfaces.ts

```diff
@@ -1,45 +1,46 @@
-import { STARTER, INTERMEDIATE, ADVANCED, } from './shared'
```

Comment:
Nothing wrong with the change here, but we could also just add `ELL_*` constants to `shared` and keep those defined there.  I don't have strong feelings either way.